rotkehlchen/__main__.py

```diff
@@ -10,11 +10,14 @@ def main():
     from rotkehlchen.server import RotkehlchenServer
     try:
         rotkehlchen_server = RotkehlchenServer()
-    except: # noqa
-        tb = traceback.format_exc()
-        logging.critical(tb)
-        print("Failed to start rotkehlchen backend:\n{}".format(tb))
-        sys.exit(1)
+    except SystemExit as e:
```

Comment:
It's duplicating code but you would need to also leave the previous except there.

So final result should be like:

```python
except SystemExit as e:
    if e.code is None or e.code == 0:
        sys.exit(0)
    else:
        tb = traceback.format_exc()
        logging.critical(tb)
        print("Failed to start rotkehlchen backend:\n{}".format(tb))
        sys.exit(1)
except: # noqa
    tb = traceback.format_exc()
    logging.critical(tb)
    print("Failed to start rotkehlchen backend:\n{}".format(tb))
    sys.exit(1)
```